### DISCUSSION TITLE
cannot merge my PR in my repository

### DISCUSSION THREAD

**Discussion by Delta456** [community/community#37772] [Pull Requests]
I want to merge my [PR](https://github.com/Delta456/box-cli-maker/pull/22) into my [repository](https://github.com/Delta456/box-cli-maker) but it seems to tell me that I need to approve my own PR first.

![image](https://user-images.githubusercontent.com/28479139/198873311-2306da75-d0f7-4dc5-8f7e-bffb2df7e689.png)

But when I approve my PR, it says PR authors cannot review their own PR but I am the author of the repository...  

![image](https://user-images.githubusercontent.com/28479139/198873362-e0388856-eaf5-427d-b9d3-70b027b1b05b.png)

I have tried changing the settings but it doesn't seem to work for me.

**Comment by DSDmark** [comment]
Did you check beach protection rule?

**Reply by Delta456** [reply]
I did check it out and it doesn't seem to work for me.

**Comment by airtower-luna** [comment]
If your branch protection rule requires a review, the review must be from someone else. To merge your PR without that review you'll have to either remove the review requirement from the branch protection rule, or give yourself the right to override it.
